src/ansys/hps/client/jms/resource/task_definition.py

```diff
@@ -156,6 +173,31 @@ def __init__(self, name: str = missing, version: str = missing, **kwargs):
 SoftwareSchema.Meta.object_class = Software
 
 
+class WorkerContext(Object):
+    """Provides the worker context resource.
+
+    Parameters
+    ----------
+    max_runtime : int, optional
```

Comment:
Usually, when "optional" is stated, we have a line immediately after the brief description like this: "The default is X." Some libraries specify the default like this immediately after the data type, but I don't know if yours supports this.

max_runtime : int, default: 30

Reply:
Doesn't apply here. Optional fields are defaulted server-side, most often to null.